src/app/components/profile/profile.controller.js

```diff
@@ -5,13 +5,31 @@
         .module('xyz-cv-ui.profile')
         .controller('ProfileController', ProfileController);
 
-        function ProfileController() {
+        function ProfileController(Skill) {
             var vm = this;
 
             vm.hello = [6, 7, 8, 9, 10];
+            vm.skills = [];
+
+            getSkills();
 
             //////////////
 
+            function getSkills() {
+                vm.skills = Skill.query();
+            }
+
+            function createSkill() {
+                var skill = {
+                    name: 'testSkill'
+                };
+                Skill.save(skill);
```

Comment:
I realize this is only test code, but in the future, with real code, I'd prefer `skill.$save()` instead of `Skill.save(skill)`